struts2-jquery-plugin/src/main/resources/template/jquery/autocompleter.ftl

```diff
@@ -18,27 +18,40 @@
  * under the License.
  */
 -->
+<#assign hasFieldErrors = parameters.widgetname?? && fieldErrors?? && fieldErrors[parameters.widgetname]??/>
+<input type="hidden"
+    <#if parameters.widgetid?if_exists != "">
+        id="${parameters.widgetid?html}"<#rt/>
+    </#if>
+    <#if parameters.nameValue??>
+        value="<@s.property value="parameters.nameValue"/>"<#rt/>
+    </#if>
+    <#if parameters.widgetname?if_exists != "">
+        name="${parameters.widgetname?html}"<#rt/>
+    </#if>
+    <#if parameters.cssClass?has_content && !(hasFieldErrors && parameters.cssErrorClass??)>
+        class="${parameters.cssClass?html}"<#rt/>
+    <#elseif parameters.cssClass?has_content && (hasFieldErrors && parameters.cssErrorClass??)>
+        class="${parameters.cssClass?html} ${parameters.cssErrorClass?html}"<#rt/>
+    <#elseif !(parameters.cssClass?has_content) && (hasFieldErrors && parameters.cssErrorClass??)>
+        class="${parameters.cssErrorClass?html}"<#rt/>
+    </#if>
+    <#if parameters.cssStyle?has_content && !(hasFieldErrors && (parameters.cssErrorStyle?? || parameters.cssErrorClass??))>
+        style="${parameters.cssStyle?html}"<#rt/>
+    <#elseif hasFieldErrors && parameters.cssErrorStyle??>
+        style="${parameters.cssErrorStyle?html}"<#rt/>
+    </#if>
+    <#if parameters.disabled?default(false)>
+        disabled="disabled"<#rt/>
+    </#if>
+/>

```

Comment:
> Would this be something a test could be added for (inside the struts2-jquery-integration-tests module). How do i easily reproduce this problem and see that it's fixed? Would trying to access a non-existing url be sufficient?

Yes, Tested by modifying autocompleters in the showcase app with wrong remote URL, and the error topics are fired. I thinked about updating IT, but I wanted to have a look how it's working cause I'm not familiar with selenium/PhantomJS.

> I'm not a fan of commented old versions of code.

Me too, I forgot to remove this comment ! :-(

> the controlheaders will be added below the input field

You're right, my mistake !

Concluding, I'll soon submit a commit to correct these points, maybe with a test case for IT as well.